server/series/utils.py

```diff
@@ -141,15 +145,42 @@ def register_player(
 
     return SeriesRegistration.objects.create(series=series, team=team, player=player), None
 
+def generate_invitation_token(
+    invitation_id: int
+) -> str:
+    mail_secret_key = settings.EMAIL_SECRET_KEY
+
+    payload = {
+        "invitation_id": invitation_id,
+    }
 
-def send_invitation_email(from_user: User, to_player: Player, team: Team, series: Series) -> None:
+    token = jwt.encode(payload, mail_secret_key, algorithm="HS256")
+    return token
+
+def get_details_from_invitation_token(
+    token: str
+) -> tuple[bool, Optional[int]]:
+    try:
+        mail_secret_key = settings.EMAIL_SECRET_KEY
+        payload = jwt.decode(token, mail_secret_key, algorithms=["HS256"])
+
+        return True, payload["invitation_id"]
```

Comment:
Here, can probably add a check to see if `invitation_id` exists in the payload. The links we generate will always have the id, it would always exist because we generate the payload and encode with the secret key, but just in case